packages/augur-ui/src/modules/common/constants.ts

```diff
@@ -838,3 +840,17 @@ export const YES_NO_OUTCOMES = [
     isTradable: true,
   },
 ];
+
+export const SCALAR_OUTCOMES = [
+  {
+    id: 0,
+    description: 'Invalid',
+    isTradable: true,
+  },
+  {
+    id: 1,
+    description: NON_EXISTENT,
```

Comment:
this should be scalarDenomination on the marketInfo object

Reply:
it gets replaced. but need to put something there in the meantime, it could be ""

Reply:
works for me